src/NetMQ/Msg.cs

```diff
@@ -265,6 +265,18 @@ public Span<byte> Slice()
             
             return new Span<byte>(m_data, m_offset, Size);  
         }
+        
+        /// <summary>
+        /// Return the internal buffer as Memory
+        /// </summary>
+        /// <returns>The memory</returns>
+        public Memory<byte> SliceAsMemory()
```

Comment:
It's a shame We didn't use `AsSpan` instead of `Slice`, so this could be `AsMemory` too.

Reply:
Yes, it's ok or you want to rename it?